react/__snapshots__/examples.spec.jsx.snap

```diff
@@ -2051,6 +2051,10 @@ exports[`DateMonthPicker should render examples: DateMonthPicker 1`] = `
 </div>"
 `;
 
+exports[`Dialog should render examples: Dialog 1`] = `"<div><button>Toggle modal</button></div>"`;
```

Comment:
on devrait mettre un open={isTesting} ou un truc du genre pour snapshoter le composant

Reply:
Bah en fait… il y est déjà. Mais malgré ca le snapshot n'est pas dans le composant, même en forçant le `open` a `true`, je ne comprend pas pourquoi.

Reply:
https://github.com/cozy/cozy-ui/issues/1552